### DISCUSSION TITLE
Will it work with vue-demi?

### DISCUSSION THREAD

**Discussion by jens-morten-mikkelsen** [histoire-dev/histoire#140] [Q&A] [answered]
I'm currently working on a project where we are running a vite setup with vue2 using vue-demi.
The plan is to either move to vue2.7 when released or move to vue3 when/if we no longer need IE11 support.

I am however curious if Histoire will work on the project (i haven't testet yet), because we are using vue-demi which has both vue2 and vue3, so maybe it can run Histoire using vue-demi/vue3 but our normal dev using vue-demi/vue2?

im open for input before i start testing :)

**Comment by Akryum** [comment, marked as answer]
I guess you'll probably have some trouble installing both vue 2 and vue 3

**Reply by Akryum** [reply]
Once #87 is done it'll be easier to support Vue 2.

**Reply by jens-morten-mikkelsen** [reply]
okay, i guess you're right, and great to see that vue2 could be easier in the future :D
